Change explanation of environment variables for ServiceBus

cjthompson opened the pull request:
`/service-bus-nodejs-how-to-use-topics-subscriptions.md`

Change the article to suggest using the variable `AZURE_SERVICEBUS_CONNECTION_STRING`.

If a previous step, the developer is told how to obtain the connection string but later is told to use two different environment variables and now how to parse the connection string into those variables.

`servicebussettings.js` supports obtaining the entire connection string from the environment variable `AZURE_SERVICEBUS_CONNECTION_STRING`.  Changing the documentation to suggest using this variable makes sense given the instructions earlier in the article explain how to obtain the entire connection string.

See https://github.com/Azure/azure-sdk-for-node/blob/master/lib/common/lib/services/servicebussettings.js#L244 : 

```js
  } else if (process.env[EnvironmentVariables.AZURE_SERVICEBUS_CONNECTION_STRING]) {
    connectionString = process.env[EnvironmentVariables.AZURE_SERVICEBUS_CONNECTION_STRING];
```

azurecla commented:
Hi __@gbellmann__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

rmca14 commented:
Change approved by author.

cjthompson committed:
Change explanation of environment variables for ServiceBus

In `/service-bus-nodejs-how-to-use-topics-subscriptions.md`,
change article to suggest using the variable `AZURE_SERVICEBUS_CONNECTION_STRING`
to match the earlier instructions on getting the connection string for later use.
